Makefile

```diff
@@ -1,8 +1,27 @@
+# Checks
+
 lint:
 		pylint easyfsl
 
 test:
 		pytest easyfsl
 
+isort:
+		isort easyfsl
+
+isort-check:
+		isort easyfsl --check
+
+black:
+		black easyfsl
+
+black-check:
+		black easyfsl --check
+
+mypy:
+		mypy easyfsl
```

Comment:
I've always wondered what the point of a makefile that does nothing but run a single command. I think it's the standard way at Sicara but not sure it's relevant here.

For instance, you could just drop the makefile and replace `make mypy` by `mypy easyfsl` : more explicit in the CI config and drop a useless makefile.

It's my opinion, you're the repo maintainer, it's up to you :)